### PR TITLE
Reformat git file system #14

### PR DESCRIPTION
**Pull request by chidiewenike**
Created Feature Extraction, Model, and False Activation Classes for reuse. Then made separate programs for training a model and false activation detection. Then reformatted the file system to add all scripts to a singular directory (Scripts). Lastly, create a JSON to place the path to the repo in the local file system.

TODO

- [x]  Make it readily compatible with MAC OS and Linux. 
- [x]  Create a README

### CODE REVIEW

**Comment by chidiewenike** [comment]
I will revise the audio script

File: Scripts/Audio_Data_Recording.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+# Description: Records audio and labels the recording
+# =============================================================================
+
+
+import wave
+import pyaudio
+import os
+import time
+import winsound
+
```
Comment (Jason-Ku) [inline Scripts/Audio_Data_Recording.py]: Looks like this file need some style formatting - I can handle it once this is merged in

Comment (Jason-Ku) [reply Scripts/Audio_Data_Recording.py]: Everything looks good otherwise

**Comment by chidiewenike** [comment]
Closes #14

**Comment by chidiewenike** [comment]
Close #10

**Comment by chidiewenike** [comment]
Close #12

**Comment by chidiewenike** [comment]
Close #16

**Comment by chidiewenike** [comment]
Removed the excess files

**Pull request merged**